### ISSUE TITLE
track width not update on content change

### ISSUE THREAD

**Issue by devel0** [g12i/react-infinite-ticker#8]
Hi, thx for this nice project,

I observed a behavior for my use case, I have a text horizontal scrolling, then after some time a change the text then the width of the child I inserted in the HorizontalTicker changed but seems the track not update the width accordingly and overlaps track1, track2.

A repro about this is available [here](https://github.com/devel0/repros/tree/react-infinite-ticker/scrolled-width-change-speed)

```sh
git clone https://github.com/devel0/repros
cd repros
git checkout react-infinite-ticker/scrolled-width-change-speed
npm i
npm run start
```

to see that observe the third line, initially it has the same text as first line, then after 5 sec I switch to the same text of line2 ( that is different in width ) ; when track1, track2 changes you notice following

![image](https://github.com/user-attachments/assets/3e711de5-d8f4-4abe-a418-f7a99e949ecf)

note that "END.ge from test1" should instead "END.BEGIN..." like the 2th line
Seems it not considered the part in red rectangle below because not updated the track width I suppose

![image](https://github.com/user-attachments/assets/1fdbd3ff-bed5-42f3-b32d-027df8cc5613)

UPDATE: Added a possible workaround [here](https://github.com/devel0/repros/commit/10374c2bac02cb422738446cdb96aad33a18e2f7)

**Comment by g12i** [comment]
Thanks @devel0 I'll take a look in the afternoon

**Comment by g12i** [comment]
Hi @devel0  I published 2.2.0, please see with following fixes https://github.com/g12i/react-infinite-ticker/pull/9 
Hopefully this solves your issue

**Comment by devel0** [comment]
Hi g12i, tried 2.2.0 and it works like a charm.
Thanks you.
